deploy/templates/notificationtemplates/appstudio/userdeactivated/notification.html

```diff
@@ -41,11 +41,11 @@
 >
 
     <p>
-        You are receiving this email because you have an account associated with {{.UserEmail}}.
+        You are receiving this email because you have a Red Hat Trusted Application Pipeline account associated with {{.UserEmail}}.
     </p>
 
     <p>
-        Your account is now deactivated and all your data has been deleted. You can request new access by signing up again at {{.RegistrationURL}}
+        Your RHTAP account is now deactivated and all your RHTAP data has been deleted. You can request new access by signing up again at {{.RegistrationURL}}
```

Comment:
```suggestion
        Your RHTAP account is now deactivated and all your application data has been deleted. You can request new access by signing up again at {{.RegistrationURL}}
```
I see this suggestions in one of the comments on the jira ticket : https://issues.redhat.com/browse/RHTAPBUGS-295
 since : _doesn't make it clear that their user data wasn't deleted (that takes a GDPR request), only their RHTAP application data was deleted._

Reply:
Right - I misread the comment in the Jira. Good catch, thanks :+1: